src/test/kotlin/com/zepben/evolve/services/network/testdata/HvEquipmentBelowLvFeederHeadNetwork.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.services.network.testdata
+
+import com.zepben.evolve.cim.iec61970.base.core.BaseVoltage
+import com.zepben.evolve.services.network.NetworkService
+import com.zepben.evolve.testing.TestNetworkBuilder
+
+object HvEquipmentBelowLvFeederHeadNetwork {
+
+    //
+    // - or |: LV line
+    // = or #: HV line
+    //
+    //      c1     c3     c7         c9     c10
+    // b0 ------+------+------ tx8 ======+------
+    //          #      |
+    //       c2 #   c4 |
+    //          #      |
+    //                tx5
+    //                 #
+    //              c6 #
+    //                 #
+    //
+    // lvf11 head terminal is b0-t2
+    //
+    fun create(): NetworkService {
+        val hvBaseVoltage = BaseVoltage().apply { nominalVoltage = 1000 }
+        val lvBaseVoltage = BaseVoltage().apply { nominalVoltage = 999 }
```

Comment:
suggestion: Suggest using realistic HV and LV voltages as 1000 is just an abitrary values that is halfway between them, rather than a real "break point". LV is typically 415/240 in Australia, 190/110 in the US (and many other values around these for other countries), with 11000/6350 as the lowest of the MV/HV voltages.

This also applies to other tests.